README.md

```diff
@@ -1,17 +1,17 @@
-#waitForImages#
+# waitForImages
```

Comment:
Thanks, I had been meaning to do this as it wasn't compatible with npmjs markdown parser :D